### PR TITLE
Wishlist

### PR DESCRIPTION
**Pull request by Megha-Joshi**
Screenshot (78)

This is the wishlist page of my E-commerce website.

### CODE REVIEW

File: css/navbar.css [hunk 1]
```diff
@@ -54,8 +47,9 @@
     color: var(--bg-color);
     font-size: 16px;
```
Comment (itsmebipul00) [inline css/navbar.css]: 1rem ??

File: css/product-wishlist.css [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+.card-head{
+    text-align: center;
+}
+.card-text{
+    text-align: center;
+    color: var(--accent-color);
+}
+.card-img{
+    width: 400px;
+    height: 300px;
```
Comment (rutu14) [inline css/product-wishlist.css]: Unit Change - This can be in rems.

File: css/root.css [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    color: var(--accent-hover);
+}
+
+.no-margin {
+    margin: 0px;
+}
+
+.sm-text {
+    font-weight: 300;
+    font-size: 16px;
```
Comment (itsmebipul00) [inline css/root.css]: 1 rem ??

File: css/signup-login.css [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+.btn-no-bg:hover{
+    color: var(--accent-hover);
+}
+.new-ac{
+    width: 100%;
+    text-align: center;
+    margin-top: 5px;
+}
+.login-head{
+    margin: 10px 20px 20px 20px;
```
Comment (itsmebipul00) [inline css/signup-login.css]: You could use rems here

File: style.css [hunk 5]
```diff
@@ -1,66 +1,72 @@
...
+
+.img-category {
     display: flex;
     flex-direction: column;
     align-items: center;
 }
-.hero-image{
+
+.hero-image {
     margin-bottom: 20px;
```
Comment (itsmebipul00) [inline style.css]: Rems preferred

File: style.css [hunk 6]
```diff
@@ -1,66 +1,72 @@
...
+    margin-top: 20px;
+}
+
+.card-text {
+    color: var(--accent-color);
+    margin: 10px;
+}
+
+.res-image {
+    height: 450px;
```
Comment (rutu14) [inline style.css]: Unit Change - This can be in rems.

File: style.css [hunk 7]
```diff
@@ -1,66 +1,72 @@
...
+.hz-card {
     flex-basis: 30rem;
     height: 40%;
     padding: 10px;
-    justify-content: center;
-    align-items: center; 
+}
+
+.card-head {
+    margin-top: 20px;
```
Comment (rutu14) [inline style.css]: Unit Change - This can be in rems.

File: style.css (hunk 6 above)
Comment (itsmebipul00) [inline style.css]: Rems ??

**Review by rutu14 — APPROVED** [review]

**Review by itsmebipul00 — APPROVED** [review]
Good to merge with suggested changes

**Pull request merged**